apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -78,10 +85,28 @@ export default class TutorialExplorer extends React.Component {
       showingModalFilters: false,
       sortBy: sortBy,
       orgName: orgName,
-      showingAllTutorials: showingAllTutorials
+      showingAllTutorials: showingAllTutorials,
+      searchTerm: defaultSearchTerm
     };
   }
 
+  handleSearchTerm = searchTerm => {
+    const filteredTutorials = this.filterTutorialSet(
+      this.state.filters,
+      this.state.sortBy,
+      this.state.orgName,
+      searchTerm
+    );
+
+    this.setState({
+      searchTerm,
+      filteredTutorials,
+      filteredTutorialsCount: filteredTutorials.length
+    });
+
+    this.scrollToTop();
```

Comment:
Not sure whether it's specifically this call, but after typing in the search box and then immediately scrolling the page, the scroll position jumps around unexpectedly.  I think it's this scroll action competing with my own scrolling.